src/commands/fun/latex.ts

```diff
@@ -0,0 +1,86 @@
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = `Accepts a LaTeX expression and posts it as a rendered image.`;
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'input',
+			description: 'The LaTeX expression to render',
+			type: 'STRING',
+			required: true
+		},
+		{
+			name: 'transparent',
+			description: 'Whether to render the image with a transparent background (default: false)',
+			type: 'BOOLEAN',
+			required: false
+		}
+	]
+
+	async tempRun(interaction: CommandInteraction): Promise<void> {
+		// input should never be null
+		const tex = interaction.options.getString('input') ?? '';
+		const requestURL = 'https://latex.codecogs.com/svg.json?' + tex;
+		const errorResponse = "Sorry, I couldn't render that LaTeX expression.";
+		let imageAsBase64: Buffer;
+		try {
+			const response = await fetch(requestURL, { method: 'Get' });
+			const imageAsBase64JSON = await response.json();
+			imageAsBase64 = Buffer.from(imageAsBase64JSON["latex"]["base64"], 'base64');
+		} catch (error) {
+			return interaction.reply({ content: errorResponse, ephemeral: true });
+		}
+
+		const image = await loadImage(imageAsBase64);
+
+		// Image will have 4 pixels of padding on all sides
+		const padding = 4;
+		const canvasWidth = image.width + (padding * 2);
+		const canvasHeight = image.height + (padding * 2);
+
+		const canvas = createCanvas(canvasWidth, canvasHeight);
+		const ctx = canvas.getContext('2d');
+		ctx.clearRect(0, 0, canvasWidth, canvasHeight);
+
+		const transparentBg = interaction.options.getBoolean('transparent') ?? false;
+		if (!transparentBg) {
+			// Invert the default Discord embed background color, entire canvas is inverted later
+			ctx.beginPath();
+			ctx.fillStyle = 'rgb(' + String(255 - 47) + ',' + String(255 - 49) + ',' + String(255 - 54) + ')';
+			ctx.fillRect(0, 0, canvasWidth, canvasHeight);
+		}
+
+		// Draw image and invert color - necessary because the text is black and unreadable by default
+		ctx.drawImage(image, padding, padding);
+		try {
+			const canvasData = ctx.getImageData(0, 0, canvasWidth, canvasHeight);
+
+			// https://stackoverflow.com/a/70097101/7545722
+			// no other solution works here? this is a bit slower than the other methods but barely noticeable,
+			// especially considering the size of the average image is very small
+			// and this keeps transparency
+			for (let i = 0; i < canvasData.data.length; i += (i % 4 === 2 ? 2 : 1)) {
+				canvasData.data[i] = 255 - canvasData.data[i];
+			}
+			ctx.putImageData(canvasData, 0, 0);
+		} catch (error) {
+			return interaction.reply({ content: errorResponse, ephemeral: true });
+		}
+
+		const file = new MessageAttachment(canvas.toBuffer(), 'tex.png');
+
+		const embed = new MessageEmbed()
+			.setImage('attachment://tex.png');
+
+		return interaction.reply({ embeds: [embed], files: [file] });
+	}
+
+	async run(): Promise<void> {
+		return;
+	}
```

Comment:
Add an empty line (with no whitespace) below this closing bracket and the one on line 86.